common-lib/src/cmr/common/log.clj

```diff
@@ -49,7 +75,30 @@
   ;; Set the format for logging.
   (t/merge-config!
     {:output-fn log-formatter
-     :appenders {:println {:enabled? stdout-enabled?}}}))
+     :appenders {:println {:enabled? stdout-enabled?}}})
+
+  ;; this is to be able to set a log level by namespace at run time
+  (t/merge-config! {:middleware [(partial log-by-ns-pattern)]})
+  (t/merge-config! {:ns-pattern-map {:all level}}))
+
+;; The following 2 functions are to implement run-time log changes.
+(defn get-logging-configuration
+  "This function gets the logging configuration to the caller as a map."
+  []
+  t/*config*)
+
+(defn merge-logging-configuration
+  "This function merges the passed in configuration with the current logging configuration.
+  The passed in configuration map includes a map of name spaces with logging levels and at the end
+  use :all and its level to signify what all the other non identified name spaces' logging level
+  should be.
+  An example of a passed in configuration is as follows
+  { :level :info
```

Comment:
looks like there's a space here between the `{` and the `:level` ...?

Reply:
yes, I have removed it.